components/clientComponents/forms/AddressComplete/types.ts

```diff
@@ -0,0 +1,42 @@
+"use client";
+import { InputFieldProps } from "@lib/types";
+
+// AutoComplete API returns 1 or more objects like:
+export interface AddressCompleteChoice {
+  Id: string;
+  Text: string;
+  Highlight: string;
+  Description: string;
+  IsRetrievable: boolean;
+}
+
+// Address Lookup API returns an array of objects like:
+export interface AddressCompleteResult {
+  FieldGroup: string;
+  FieldName: string;
+  FormattedValue: string;
+  FieldType: string;
+  FieldSequence: number;
+}
+
+// Props for the AddressComplete component
+export interface AddressCompleteProps extends InputFieldProps {
```

Comment:
Same type different name?

https://github.com/cds-snc/platform-forms-client/pull/4197/files#diff-59ba237409062af9738110bb0f8018b8a37719a559cc4e8c2dca1abf620a2006R37

Reply:
One is used for the Input during customization of the component, the other is output which won't have the InputFieldProps. Other components had their type in the Responses/Review area, but I kept it in the component.